usb.cpp

```diff
@@ -26,17 +26,13 @@ extern "C"
     #undef LOG_DEBUG
     #undef LOG_INFO
     #undef LOG_WARN
-    #undef LOG_ERROR
-    #undef LOG_FATAL
     #undef LOG_CUSTOM
 
     #define LOG_PRINTF(...)
     #define LOG_TRACE(...)
     #define LOG_DEBUG(...)
     #define LOG_INFO(...)
     #define LOG_WARN(...)
-    #define LOG_ERROR(...)
```

Comment:
@mjan-dev @swichu91 I think errors should always be logged. What do you think about it?

Reply:
Please check out the logging layer I recently added. You might find it useful.
`third-party/usb_stack/log.hpp`

Reply:
Ok, I changed it!